src/SeeMore/SeeMore.js

```diff
@@ -136,14 +141,10 @@ class SeeMore extends React.Component {
 
     return (
       <Text
-        testID="SeeMore"
         onLayout={isShowingMore ? undefined : this.onLayout}
         numberOfLines={isShowingMore ? undefined : numberOfLines}
-        {...this.panResponder.panHandlers}
```

Comment:
Same here, seems like this is an unexpected change

Reply:
Fixed ✅

ps. It's prettier :)